tests/pytests/test.py

```diff
@@ -975,6 +975,20 @@ def testSlopInOrder(env):
     env.assertEqual(0, r.execute_command(
         'ft.search', 'idx', 't1 t2 t3 t4', 'inorder')[0])
 
+    # test with qsort optimization on intersect iterator
```

Comment:
I also suggest creating another test for it (maybe with the issue number).
When tests get large it's harder to spot what went wrong when they failed.